atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -475,7 +476,14 @@ class ReactorConfig(object):
     Class to parse the atomic-reactor configuration file
     """
 
-    DEFAULT_CONFIG = {ReactorConfigKeys.VERSION_KEY: 1}
+    DEFAULT_CONFIG = {
+        ReactorConfigKeys.VERSION_KEY: 1,
+        ReactorConfigKeys.KOJI_KEY: {
```

Comment:
Why is this needed?

Reply:
Some tests are using 'DEFAULT_CONFIG' to configure reactor_config_map. 
Inasmuch as 'koji' is mandatory, I've decided to add 'koji' into 'DEFAULT_CONFIG'.

Reply:
Fix tests, don't modify code with random default values because tests are failing

Reply:
Fixed